Demo/Demo/SwiftUIComponents/CardPaymentViews/CardPaymentOrderCompletionView.swift

```diff
@@ -6,21 +6,33 @@ struct CardPaymentOrderCompletionView: View {
     @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
 
     var body: some View {
-        VStack {
-            CardApprovalResultView(cardPaymentViewModel: cardPaymentViewModel)
-            if cardPaymentViewModel.state.approveResult != nil {
-                CardOrderActionButton(
-                    intent: cardPaymentViewModel.state.intent,
-                    orderID: orderID,
-                    selectedMerchantIntegration: DemoSettings.merchantIntegration,
-                    cardPaymentViewModel: cardPaymentViewModel
-                )
-            }
+        let state = cardPaymentViewModel.state
```

Comment:
Do we need to set this as a variable? Might be more clear to just reference `cardPaymentViewModel.state` when needed

Reply:
It was referencing Jax's feedback, that there were multiple references to this and that it might be more readable to have a simpler variable.